src/diffusers/models/attention.py

```diff
@@ -243,25 +264,42 @@ def forward(
         timestep=None,
         attention_mask=None,
         cross_attention_kwargs=None,
+        class_labels=None,
     ):
+        if self.use_ada_layer_norm:
+            scale, shift = self.norm1(timestep)
+            norm_hidden_states = self.norm1.norm(hidden_states)
```

Comment:
We should avoid `norm1.norm(...)` -> it makes it quite difficult to read the code. 
Happy to help with the refactor afterwards though!

Reply:
right good question... we were just checking what was needed but i think i can try to put it back and simplify

Reply:
I have reverted it to the original API so there should be no change